aws_demo_project.yml

```diff
@@ -1,33 +1,36 @@
-schema_version: 0.2
-stock_type: residential
+schema_version: '0.3'
 buildstock_directory: ../resstock  # Relative to this file or absolute
 project_directory: project_national  # Relative to buildstock_directory
 output_directory: ../demo_test_outputs
 weather_files_url: https://data.nrel.gov/system/files/128/ResStock_TMY3.zip
```

Comment:
Update url to https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip to be compatible with resstock 2.4.0.